contracts/aggregator/fantom/flashloan/implFLA/main.sol

```diff
@@ -0,0 +1,77 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+import "../helpers.sol";
+
+contract FLAImplementationFantom is Helper {
+
+    /**
+     * @dev Main function for flashloan for all routes. Calls the middle functions according to routes.
+     * @notice Main function for flashloan for all routes. Calls the middle functions according to routes.
+     * @param _tokens token addresses for flashloan.
+     * @param _amounts list of amounts for the corresponding assets.
+     * @param _route route for flashloan.
+     * @param _data extra data passed.
+    */
+    function flashLoan(	
+        address[] memory _tokens,	
+        uint256[] memory _amounts,
+        uint256 _route,
+        bytes calldata _data,
+        bytes calldata // kept for future use by instadapp. Currently not used anywhere.
+    ) external reentrancy {
+
+        require(_route == 10, "invalid-FLA-route");
+        (_tokens, _amounts) = bubbleSort(_tokens, _amounts);
+        validateTokens(_tokens);
+        routeFLA(msg.sender, _tokens, _amounts, _data);
+    }
+
+    function routeFLA(
+        address _receiverAddress,
+        address[] memory _tokens,
+        uint256[] memory _amounts,
+        bytes memory _data
+    ) internal returns (bool) {
+
+        FlashloanVariables memory instaLoanVariables_;
+        instaLoanVariables_._tokens = _tokens;
+        instaLoanVariables_._amounts = _amounts;
+        instaLoanVariables_._instaFees = calculateFees(
+            _amounts,
+            calculateFeeBPS(10)
+        );
+        instaLoanVariables_._iniBals = calculateBalances(
+            _tokens,
+            address(this)
+        );
+
+        safeTransfer(instaLoanVariables_, _receiverAddress);
+
+        if (checkIfDsa(_receiverAddress)) {
+            Address.functionCall(
+                _receiverAddress,
+                _data,
+                "DSA-flashloan-fallback-failed"
+            );
+        } else {
+            require(InstaFlashReceiverInterface(_receiverAddress).executeOperation(
+                _tokens,
+                _amounts,
+                instaLoanVariables_._instaFees,
+                _receiverAddress,
+                _data
+            ), "invalid flashloan execution");
+        }
+
+        instaLoanVariables_._finBals = calculateBalances(
+            _tokens,
+            address(this)
+        );
+
+        validateFlashloan(instaLoanVariables_);
+
+        status = 1;
+        delete(implToCall);
```

Comment:
@shriyatyagii I think I am seeing the `delete` keyword used like this for the first time. I remember it will be something like this:
```solidity
delete implToCall;
```

If it's working fine, then cool

Reply:
Tried consoling, `delete` works fine like this. Tho I think `delete implToCall` is the proper syntax. I've updated.